vllm/core/scheduler.py

```diff
@@ -1,6 +1,7 @@
 import enum
 import time
 from typing import Dict, Iterable, List, Optional, Tuple, Union
+from collections import deque
```

Comment:
```suggestion
from collections import deque
import enum
import time
from typing import Dict, Iterable, List, Optional, Tuple, Union
```
style nit: Please alphabetically sort the import statements 😄